src/monarch_py/cli.py

```diff
@@ -93,6 +94,7 @@ def associations(
         object: The object of the association
         entity: The subject or object of the association
         limit: The number of associations to return
+        direct: Whether to only return associations with subject/object as ancestors
```

Comment:
It's the reverse of this, direct means excluding ancestors, so associations where the subject or object on the association is directly equal to the ID specified